spark-doris-connector/src/test/java/org/apache/doris/spark/rest/TestRestService.java

```diff
@@ -44,14 +44,13 @@
 import org.apache.doris.spark.rest.models.Schema;
 import org.apache.doris.spark.rest.models.Tablet;
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import jdk.nashorn.internal.ir.annotations.Ignore;
```

Comment:
JDK11 removed

Reply:
@cxzl25   Add me WeChat: 35926237, there is a problem with your compilation